cmd/security-bootstrapper/entrypoint.sh

```diff
@@ -0,0 +1,221 @@
+CONSUL_READY_PORT_KEY="ConsulReadyPort"
+POSTGRES_HOST_KEY="PostgresHost"
+POSTGRES_PORT_KEY="PostgresPort"
+POSTGRES_READY_PORT_KEY="PostgresReadyPort"
+REDIS_HOST_KEY="RedisHost"
+REDIS_PORT_KEY="RedisPort"
+REDIS_READY_PORT_KEY="RedisReadyPort"
+PROXY_SETUP_HOST_KEY="ProxySetupHost"
+bootstrapper_host=""
+bootstrap_port_number=0
+vaultworker_host=""
+tokens_ready_port_number=0
+ready_to_run_port_number=0
+consul_host=""
+consul_port_number=0
+consul_ready_port_number=0
+postgres_host=""
+postgres_port_number=0
+postgres_ready_port_number=0
+redis_host=""
+redis_port_number=0
+redis_ready_port_number=0
+proxy_setup_host=""
+
+while read -r key value
+do
+  trimmed_key=$(trim_spaces "${key}")
+  trimmed_value=$(trim_spaces "${value}")
+
+  if [ "${trimmed_key}" = "${BOOTSTRAPPER_HOST_KEY}" ]; then
+    bootstrapper_host=${trimmed_value}
+  elif [ "${trimmed_key}" = "${BOOTSTRAP_PORT_KEY}" ]; then
+    bootstrap_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${VAULT_WORKER_HOST_KEY}" ]; then
+    vaultworker_host=${trimmed_value}
+  elif [ "${trimmed_key}" = "${TOKENS_READY_PORT_KEY}" ]; then
+    tokens_ready_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${READY_TO_RUN_PORT_KEY}" ]; then
+    ready_to_run_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${CONSUL_HOST_KEY}" ]; then
+    consul_host=${trimmed_value}
+  elif [ "${trimmed_key}" = "${CONSUL_PORT_KEY}" ]; then
+    consul_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${CONSUL_READY_PORT_KEY}" ]; then
+    consul_ready_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${POSTGRES_HOST_KEY}" ]; then
+    postgres_host=${trimmed_value}
+  elif [ "${trimmed_key}" = "${POSTGRES_PORT_KEY}" ]; then
+    postgres_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${POSTGRES_READY_PORT_KEY}" ]; then
+    postgres_ready_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${REDIS_HOST_KEY}" ]; then
+    redis_host=${trimmed_value}
+  elif [ "${trimmed_key}" = "${REDIS_PORT_KEY}" ]; then
+    redis_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${REDIS_READY_PORT_KEY}" ]; then
+    redis_ready_port_number=${trimmed_value}
+  elif [ "${trimmed_key}" = "${PROXY_SETUP_HOST_KEY}" ]; then
+    proxy_setup_host=${trimmed_value}
+  fi
+done < ${TOML_FILE}
+
+echo bootstrapper_host: "${bootstrapper_host}" vaultworker_host: "${vaultworker_host}" postgres_host: "${postgres_host}"
+echo consul_host: "${consul_host}" redis_host: "${redis_host}" proxy_setup_host: "${proxy_setup_host}"
+echo bootstrap_port_number: "$bootstrap_port_number" tokens_ready_port_number: "$tokens_ready_port_number"
+echo ready_to_run_port_number: "$ready_to_run_port_number" postgres_port_number: "$postgres_port_number"
+echo consul_ready_port_number: "$consul_ready_port_number" redis_ready_port_number: "$redis_ready_port_number"
+
+EDGEX_VAULT_HOST="edgex-vault"
+EDGEX_VAULT_PORT=8200
+KONG_HOST="kong"
+KONG_STATUS_PORT=8001
```

Comment:
These should be environment variables set in common-security.env in Compose Builder.

Reply:
added